tests/ui/tuple_enum.stderr

```diff
@@ -0,0 +1,42 @@
+error[E0599]: no function or associated item named `augment_clap` found for type `u32` in the current scope
```

Comment:
lack the rs file, or this file should not be added

Reply:
Indeed, good catch. Fixed.